src/commands/hgetf.json

```diff
@@ -1,6 +1,6 @@
 {
     "HGETF": {
-        "summary": "For each specified field, returns its value and optionally set the field's remaining expiration time in seconds / milliseconds",
+        "summary": "For each specified field: get its value and optionally set the field's remaining time to live / UNIX expiration timestamp in seconds / milliseconds",
         "complexity": "O(N) where N is the number of arguments to the command",
         "group": "hash",
         "since": "8.0.0",
```

Comment:
Shouldn't it be 7.4 (for all HFE commands)?

Reply:
I'm aware of this and we need to change it for all hfe commands. I'll address in a future PR.